Simplify jaffle_shop example with dbt-fal (#9)
fal-ai/jaffle_shop_with_fal: README.md

We are tasked to understand what kind of jaffles we make the most money from.

So we decided to run a [clustering algorithm](https://github.com/fal-ai/jaffle_shop_with_fal/blob/main/clustering.py) to separate the orders into 3 different clusters and then to calculate all the [revenue for each cluster](https://github.com/fal-ai/jaffle_shop_with_fal/blob/main/models/cluster_stats.sql). Once our clustering algorithm is done, we want to send a notification to a Slack channel notifying our team that the results are ready for viewing.

The [fal](https://github.com/fal-ai/fal) + [dbt-fal](https://github.com/fal-ai/fal/tree/main/adapter) combination is the perfect tool for the task at hand:

> - Using the `dbt-fal` adapter, we can iterate on and ship our clustering algorithm written in Python with any datawarehouse **right within our dbt project.**

Reviewer: ```suggestion
- `dbt-fal` adapter lets us iterate on and ship our clustering algorithm. We wrote it as a Python models that works with any data warehouse **right within our dbt project.**
```

> - Using the `fal flow` CLI command, we can send a Slack notification to notify our team.

Reviewer: ```suggestion
- `fal flow` CLI command lets us send a Slack notification, via a Python post-hook.
```

> With this combo, you won't to leave your dbt project while still bringing more capabilities to your stack.

Reviewer: ```suggestion
With this combo, you won't have to leave your dbt project and still add more capabilities to your stack.
```

### Installing Instructions:

1. Install `fal` and `dbt-fal`

```

> $ pip install fal dbt-fal[duckdb, postgres, redshift, bigquery]

Reviewer: This seems a bit ambiguous to me. Maybe we can select one adapter and let the users know that they can install others.

Author: Sounds good! I will switch it to duckdb :)

Reviewer: 👍

### Running Instructions:

> 1. Run `dbt run`

Reviewer: This part reads like the user should do both `dbt run` and `fal flow`. Is that the case or should the user choose one or the other?

Reviewer 2: Agreed

```bash
$ dbt run

> ## Runs the whole graph including the Python models

Reviewer 2: Does not run the notification, is that part of the "whole graph"?

Author: Not really, I think its good separate transformations vs non-transformations.

Reviewer 2: ```suggestion
## Runs the SQL models on the datawarehouse and Python models locally with fal
```

```

> 2. Run `fal flow` to execute the full graph including Python scripts. You can use the dbt [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators) and [much more](https://docs.fal.ai/). With `fal flow`, you will not have to run `dbt run` since fal handles the full execution.

Reviewer 2: ```suggestion
2. Run `fal flow run` to execute the full graph including Python scripts. You can use the dbt [graph selectors](https://docs.getdbt.com/reference/node-selection/graph-operators) and [much more](https://docs.fal.ai/). With `fal flow run`, you will not have to run `dbt run` since fal handles the full execution.
```

```bash
$ fal flow run
## Runs dbt run and the associated scripts, in this case a Slack notification is triggered

> ```

Reviewer: With `fal flow run` the user doesn't need to do `dbt run`, right? We can make a suggestion that they can use dbt but then the post-hooks won't run.
